website/static/js/keen.js

```diff
@@ -180,6 +180,7 @@ var KeenTracker = (function() {
             var _defaultPrivateKeenPayload = function() {
                 var payload = _defaultKeenPayload();
                 var user = window.contextVars.currentUser;
+                var ip = window.contextVars.anonymizedIP;
```

Comment:
we'll need to make this ip address available via some api/v2 endpoint if we take this approach, so other ember apps can do the same

Reply:
I thought ember apps took care of keen themselves https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/app/metrics-adapters/keen.ts
https://github.com/CenterForOpenScience/ember-osf/blob/develop/addon/metrics-adapters/keen.js